File: flight-management-system/src/app/flight.ts
import { FlightCompositeKeyDTO } from './flight-composite-key';

export class FlightDTO{
    constructor(

        public id:string,
        public flightId:FlightCompositeKeyDTO,
        public arrDate:string,
        public arrTime:string,
        public totalBusiness:number,
        public availableBusiness:number,
        public totalEconomy:number,
        public availableEconomy:number

    ){}
}

File: User/user-core/src/main/java/com/fms/user/core/domain/Flight.java
package com.fms.user.core.domain;

import lombok.*;
import org.springframework.data.mongodb.core.mapping.Document;

import javax.persistence.EmbeddedId;
import javax.persistence.Id;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
public class Flight {

    private String id;

    private FlightCompositeKey flightId;
    private String arrDate;
    private String arrTime;
    private int totalBusiness;
    private int availableBusiness;
    private int totalEconomy;
    private int availableEconomy;

}
